example-doc-win2019.md

```diff
@@ -0,0 +1,276 @@
+# Windows Server 2019
+
+## Installed Software
+### Languages and Runtimes
+- Java 1.7: build 1.7.0_262-b10
+- Java 1.8: build 1.8.0_252-b14 (default)
+- Java 11: build 11.0.7+10-LTS
+- Rust 1.43.0
+- Python 3.7.7
```

Comment:
Is it the same python from toolcache or not? The same for ruby

Reply:
Yes, it's default version from toolcache